common/lib/xmodule/xmodule/templates/problem/imageresponse.yaml

```diff
@@ -4,19 +4,33 @@ metadata:
     markdown: !!null
```

Comment:
As a followup, maybe @cptvitamin could give us some words about whether it is possible to make this type of problem accessible